File: cses/leetcode/go/3083.go
package main

func isSubstringPresent(s string) bool {
	hashset := make(map[string]bool)

	for l := 1; l < len(s); l++ {
		a := s[l-1 : l+1]
		hashset[a] = true
		b := string(s[l]) + string(s[l-1])
		if hashset[b] {
			return true
		}
	}
	return false
}

File: leetcode/python/3083.py
class Solution:
    def isSubstringPresent(self, s: str) -> bool:
        hashset = set()
        for l in range(1, len(s)):
            a = s[l-1]+s[l]
            hashset.add(a)
            b = s[l]+s[l-1]
            if b in hashset:
                return True
        return False